Transformer/Tests/ConditionFromBooleanToMongoTransformerTest.php

```diff
@@ -1,93 +0,0 @@
-<?php
```

Comment:
I don't think you should remove this test for now: the transformer is only deprecated, so it can always be used until it is definitively removed, so the test is always usefull.